db/schema.rb

```diff
@@ -249,6 +249,9 @@
     t.index ["uuid"], name: "index_courses_on_uuid", unique: true
   end
 
+  create_table "data_migrations", primary_key: "version", id: :string, force: :cascade do |t|
```

Comment:
Don't think this needs to be in here?

Reply:
Weird, shouldn't it be there due to [gem](https://github.com/DFE-Digital/register-trainee-teachers/blob/5763-allow-soft-deletion-of-providers/Gemfile#L102)

Reply:
I think we've gone back and forth about this since the it was first introduced. There is a table in prod to track them. Whether the schema for it needs to be in the source is debatable. I think we decided not to ages ago 🤷

Reply:
It just seems unnecessary to manual intervene for an auto generated thing.

`bundle exec rails db:migrate` 

Opening to the rest of the team for input @DFE-Digital/register-devs